internal/charm/hooks/hooks.go

```diff
@@ -108,7 +109,6 @@ func StorageHooks() []Kind {
 }
 
 var workloadHooks = []Kind{
-	PebbleChangeUpdated,
 	PebbleCustomNotice,
 	PebbleReady,
```

Comment:
Need `PebbleCheckFailed` and `PebbleCheckRecovered`.
See https://github.com/juju/charm/commit/c09ff56304f49b8d98af09ea3a568f8547bdfc5d

Reply:
Done & Done